stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -25,8 +25,11 @@ string pathSeparator = isWindows ? "\\" : "/";
 string pathListSeparator = isWindows ? ";" : ":";
 
 # Retrieves the absolute path from the provided location.
+# ```ballerina
+#  string absolutePath = checkpanic filepath:absolute(<@untainted> "test.txt");
```

Comment:
Shall we remove checkpanic and add correct return types

Reply:
Corrected.